erlang/atomvm/Portfile

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem      1.0
+PortGroup       github 1.0
+PortGroup       cmake 1.1
+
+github.setup    atomvm atomvm v0.6.0-alpha.0
+categories      erlang devel
+platforms       any
+maintainers     {pguyot @pguyot} openmaintainer
+license         Apache-2
+
+description     Tiny Erlang VM
```

Comment:
Please have consistent alignment throughout the Portfile if possible.  You can align the 2nd column of `description` and above to the alignment of `long_description`'s 2nd column.